### ISSUE TITLE
False negative assigning member

### ISSUE THREAD

**Issue by parched** [microsoft/pyright#4759]
**Describe the bug**
Pyright no longer gives an error when assigning an invalid attribute to `vtkInteractorStyleTerrain`

**To Reproduce**
Using pyright 1.1.297+
Python 3.10
vtk==9.2.2

```python
from vtkmodules.vtkInteractionStyle import vtkInteractorStyleTerrain


def do(interactor: vtkInteractorStyleTerrain):
    interactor.adfasdg = True
```

**Expected behavior**
As in 1.1.296, I expected 
> error: Cannot assign member "adfasdg" for type "vtkInteractorStyleTerrain"

**Comment by erictraut** [comment]
This will be addressed in the next release. Apologies for the regression. This was an existing bug that was masked previously, and it's subtle enough that it is difficult to cover in any unit tests.

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.299, which I just published. It will also be included in a future release of pylance.
